### DISCUSSION TITLE
Welcome to DB-GPT-Hub Discussions!

### DISCUSSION THREAD

**Discussion by csunny** [eosphoros-ai/DB-GPT-Hub#20] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kavita-Yadav** [comment]
Hi, May I know that DB-GPT-HUB is also tested for Clickhouse Database queries ? If not then for which databases it is already tested and how?

**Comment by charlie3go** [comment]
hi, I used the ChatGLM series of large models for fine-tuning and used the Spider dataset. It is found that the sql ability of llm has improved a little, but the dialogue ability has been seriously lost. Is there an effective way to use llm on sql fine-tuning? Thank you very much.
